pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConnectionPool.java

```diff
@@ -195,14 +203,91 @@ public void initChannel(SocketChannel ch) throws Exception {
                 cnx.ctx().close();
                 return null;
             });
+        }).exceptionally(exception -> {
+            log.warn("Failed to open connection to {} : {}", physicalAddress, exception.getClass().getSimpleName());
```

Comment:
is there any specific reason to log `exception.getClass().getSimpleName()` instead message?

Reply:
We had that already in the existing code. It was to just use "ConnectionError" type names I think